tests/foundry/client/data/documents/playlist.test-d.ts

```diff
@@ -0,0 +1,26 @@
+import { expectTypeOf } from "vitest";
+
+// @ts-expect-error - A Playlist requires name.
+new Playlist();
+
+// @ts-expect-error - A Playlist requires name.
+new Playlist({});
+
+const playlist = new Playlist({ name: "Some Playlist" });
+expectTypeOf(playlist).toEqualTypeOf<Playlist>();
+
+const playlistSound = new PlaylistSound({ name: "Some PlaylistSound" });
+
+expectTypeOf(playlist.playbackOrder).toEqualTypeOf<string[]>();
+expectTypeOf(playlist.visible).toEqualTypeOf<boolean>();
+expectTypeOf(playlist.prepareDerivedData()).toEqualTypeOf<void>();
+expectTypeOf(playlist.playAll()).toEqualTypeOf<Promise<Playlist | undefined>>();
+expectTypeOf(playlist.playNext()).toEqualTypeOf<Promise<Playlist | undefined | null>>();
+expectTypeOf(playlist.playSound(playlistSound)).toEqualTypeOf<Promise<Playlist | undefined>>();
+expectTypeOf(playlist.stopSound(playlistSound)).toEqualTypeOf<Promise<Playlist | undefined>>();
+expectTypeOf(playlist.stopAll()).toEqualTypeOf<Promise<Playlist | undefined>>();
+expectTypeOf(playlist.cycleMode()).toEqualTypeOf<Promise<Playlist | undefined>>();
+expectTypeOf(playlist.toAnchor()).toEqualTypeOf<HTMLAnchorElement>();
+
+// TODO: Modify to Playlist | null once data can be grabbed from CONFIG
+expectTypeOf(playlist.sheet).toEqualTypeOf<FormApplication | null>();
```

Comment:
This test should probably mention AppV2? I don't think we *want* to make assumptions about the correct sheet application given how extensible that can be.

Reply:
Oops... that was a copy/paste from somewhere else :(   Should I just remove it?  Wasn't sure what you meant about referring to AppV2?

Reply:
The updates to ClientDocument mean that AppV2 is now an option for `sheet`
```ts
get sheet(): FormApplication | foundry.applications.api.ApplicationV2 | null;
```

Reply:
gotcha